src/api/app/views/webui/projects/public_key/_key_dialog.html.haml

```diff
@@ -19,8 +20,9 @@
             = @project.key_info.algorithm
           %td
             = @project.key_info.keyid
-          %td
-            = DateTime.strptime(@project.key_info.expires, '%s').strftime('%F')
+          - if @project.key_info.expires
+            %td
+              = DateTime.strptime(@project.key_info.expires, '%s').strftime('%F') if project.key_info.expires
```

Comment:
Why do you have a second if condition at the end of this line? It's already caught by line 23.